.github/dependabot.yml

```diff
@@ -1,31 +1,124 @@
 version: 2
 updates:
+  - package-ecosystem: "cargo"
+    directory: "/"
+    allow:
+      - dependency-type: all
+    schedule:
+      interval: "daily"
+    labels:
+      - "cargo"
+      - "dependencies"
+    ignore:
+      - dependency-name: "codespan*"
+      - dependency-name: "libfuzzer-sys"
+      - dependency-name: "bindgen"
+    open-pull-requests-limit: 20
+
+  # Dockerfiles.
+
   - package-ecosystem: "docker"
-    directory: "/docker/"
+    directory: "/docker/ci/alpine"
     schedule:
       interval: "daily"
     labels:
       - "docker"
       - "dependencies"
 
-  - package-ecosystem: "github-actions"
-    directory: "/"
+  - package-ecosystem: "docker"
+    directory: "/docker/ci/arch"
     schedule:
       interval: "daily"
     labels:
-      - "github-actions"
+      - "docker"
       - "dependencies"
-    open-pull-requests-limit: 15
 
-  - package-ecosystem: "cargo"
+  - package-ecosystem: "docker"
+    directory: "/docker/ci/centos"
+    schedule:
+      interval: "daily"
+    labels:
+      - "docker"
+      - "dependencies"
+
+  - package-ecosystem: "docker"
+    directory: "/docker/ci/circleci"
+    schedule:
+      interval: "daily"
+    labels:
+      - "docker"
+      - "dependencies"
+
+  - package-ecosystem: "docker"
+    directory: "/docker/client"
+    schedule:
+      interval: "daily"
+    labels:
+      - "docker"
+      - "dependencies"
+
+  - package-ecosystem: "docker"
+    directory: "/docker/cluster-test"
+    schedule:
+      interval: "daily"
+    labels:
+      - "docker"
+      - "dependencies"
+
+  - package-ecosystem: "docker"
+    directory: "/docker/init"
+    schedule:
+      interval: "daily"
+    labels:
+      - "docker"
+      - "dependencies"
+
+  - package-ecosystem: "docker"
+    directory: "/docker/logstash"
+    schedule:
+      interval: "daily"
+    labels:
+      - "docker"
+      - "dependencies"
+
+  - package-ecosystem: "docker"
+    directory: "/docker/mint"
+    schedule:
+      interval: "daily"
+    labels:
+      - "docker"
+      - "dependencies"
+
+  - package-ecosystem: "docker"
+    directory: "/docker/safety-rules"
+    schedule:
+      interval: "daily"
+    labels:
+      - "docker"
+      - "dependencies"
+
+  - package-ecosystem: "docker"
+    directory: "/docker/tools"
+    schedule:
+      interval: "daily"
+    labels:
+      - "docker"
+      - "dependencies"
+
+  - package-ecosystem: "docker"
```

Comment:
n00b question - what would this update do for the Dockerfile?

Reply:
PRs will look like this:   https://github.com/rexhoffman/libra/pull/242

Reply:
Testing